io/src/main/java/org/commonjava/maven/ext/manip/io/ConfigIO.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (C) 2012 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.commonjava.maven.ext.manip.io;
+
+import org.codehaus.plexus.component.annotations.Component;
+import org.commonjava.maven.ext.manip.ManipulationException;
+import org.commonjava.maven.ext.manip.model.YamlFile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.representer.Representer;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+@Component( role = ConfigIO.class )
+public class ConfigIO
+{
+    private final Logger logger = LoggerFactory.getLogger( ConfigIO.class );
+
+    private final static String propertyFileString = "pme.properties";
+    private final static String yamlPMEFileString = "pme.yaml";
+    private final static String yamlPNCFileString = "pnc.yaml";
+
```

Comment:
Originally it was `pme.yaml` but given its usage context in PNC I thought being able to name it `pnc.yaml` might be helpful. No difference in format ... just the naming support.